app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -264,13 +267,33 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
   }
 
   fun markHomeMenuCloseDrawer() {
```

Comment:
can rename it as `markPreviousMenuItemCloseDrawer`

Reply:
Deferring to @rt4914, since he owns this file.